modules/db/src/test/scala/gem/dao/StepDaoSample.scala

```diff
@@ -1,18 +1,20 @@
 package gem.dao
 
-import gem.{Observation, Step}
+import gem.{Location, Observation, Step}
 import gem.config.InstrumentConfig
 
 import doobie.imports._
 
+import scalaz.==>>
+
 object StepDaoSample extends TimedSample {
-  type Result = List[Step[InstrumentConfig]]
+  type Result = Location.Middle ==>> Step[InstrumentConfig]
 
   val oid = Observation.Id.unsafeFromString("GS-2016A-Q-102-108")
 
   override def runl(args: List[String]): ConnectionIO[Result] =
     StepDao.selectAll(oid)
 
   override def format(r: Result): String =
-    r.mkString(", \n")
+    r.toAscList.map { case (l, s) => f"$l%10s -> $s" }.mkString(", \n")
```

Comment:
Ascending, which I think is the default if you use `toList` but I figured why not be explicit.